src/generate/generators/CoreFull.ts

```diff
@@ -120,6 +124,33 @@ export class AzCoreFullGenerator extends GeneratorBase {
                         ),
                     );
                 GenerateMetaFile(model);
+                await this.generateFullSingleAndAddtoOutput(
+                    new CliCmdletTest(this.model, false),
+                    true,
+                    true,
+                );
+                await this.generateFullSingleAndAddtoOutput(
+                    new CliCmdletTest(this.model, true),
+                    true,
+                    true,
+                );
+                await this.generateFullSingleAndAddtoOutput(
+                    new SimpleTemplate(
+                        this.model,
+                        path.join(
+                            model.AzextFolder,
+                            PathConstants.testFolder,
+                            PathConstants.cmdletFolder,
+                            PathConstants.conftestFile,
+                        ),
+                        path.join(
+                            PathConstants.templateRootFolder,
+                            PathConstants.testFolder,
+                            PathConstants.cmdletFolder,
+                            PathConstants.conftestFile + '.njx',
+                        ),
+                    ),
+                );
```

Comment:
I think the path join part can be moved into the template definition instead of each generators.

Reply:
Here SimpleTemplate don't bind to a particular template or output path. 
This kind of render is designed for those files which don't have any dynamic data, so that we don't need write a tiny render for several templates for many times. For example test/__init__.py files and this pytest configuration file.

Hi @qiaozha , @lirenhe ,
And I think the actual pain here is that we need to write generation code of one file for four times in CoreFull, CoreIncre, ExtensionFull, ExtensionIncre... I think it will be better to have some envelop in GeneratorBase. 
I can refactor about this if you think it's the right direction.

Reply:
I think we shuuld log a issue for that. and fix it in other PR ?

Reply:
sure https://github.com/Azure/autorest.az/issues/776